targets/bcm63xx-generic

```diff
@@ -0,0 +1,12 @@
+include 'bcm63xx.inc'
+
+device('tp-link-td-w8968-v3', 'sagem_fast-2704n', {
+	sysupgrade = '-squashfs-cfe', 
+	sysupgrade_ext = '.bin', -- No sysupgrade image available
```

Comment:
Not sure, what you mean by this.
If there wasn't a sysupgrade image available, why bother adding these lines, and how would you have tested sysupgrades were working?

Reply:
if the firmware for the device is built then for some reason there is no sysupgrade image available for the said model (its chipset) because it only creates a factory image and no sysupgrade. I did this there because normally a sysupgrade image is also generated for the target, but not with exactly this chipset